host/tests/L3_TestCases/dsHdmiIn/dsAudio_L3_Runall_Source.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+import importlib
+from pathlib import Path
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path))
+sys.path.append(os.path.join(dir_path, "../../"))
+sys.path.append(os.path.join(dir_path, "../../raft"))
+
+from raft.framework.core.logModule import logModule
+
+def Runall_L3():
+    skipTests = ["test10"]
```

Comment:
Test10 is skipped in both Sink and source RunAll file

Reply:
no for source device run all not required , will remove the file.